main.py

```diff
@@ -1,7 +1,26 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(genre="Western")
+    Genre.objects.create(genre="Action")
+    Genre.objects.create(genre="Dramma")
+
+    Actor.objects.create(first_name="George", last_name="Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+
+    Genre.objects.filter(genre="Dramma").update(genre="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(last_name="Keanu").update(last_name="Reeves")
```

Comment:
The update logic for the `Actor` model is incorrect. You should filter by `first_name='Kianu'` and update it to `first_name='Keanu'`.